rejected/process.py

```diff
@@ -272,10 +296,10 @@ def _connection_parameters(self):
             pika.PlainCredentials(
                 self.config.get('user', 'guest'),
                 self.config.get('password', self.config.get('pass', 'guest'))),
-            ssl=self.config.get('ssl', False),
+            ssl_options=self.config.get('ssl_options'),
```

Comment:
This change should be documented in docs @ https://github.com/gmr/rejected/blob/3.22/docs/configuration.rst